data/templates/telegraf/telegraf.j2

```diff
@@ -45,7 +45,7 @@
 ### Prometheus ###
 [[outputs.prometheus_client]]
   ## Address to listen on
-  listen = "{{ prometheus_client.listen_address if prometheus_client.listen_address is vyos_defined else '' }}:{{ prometheus_client.port }}"
```

Comment:
Why do we need join() if it‘s a single address?

Reply:
It appears to work fine without the join, it has been removed.